src/CrudServiceProvider.php

```diff
@@ -63,6 +65,19 @@ public function boot()
             __DIR__.'/config/backpack/crud.php',
             'backpack.crud'
         );
+
+        // Add a validation rule for "upload" field type
+        // Example of usage: 'document' => 'file_upload_crud:pdf,jpg',
+        Validator::extend('file_upload_crud', function ($attribute, $value, $parameters, $validator) {
```

Comment:
In my opinion `file` is redondant with `upload` in the rule name `file_upload_crud`, and I would have added the word `mimes`, what do you think of `crud_upload_mimes`  ?